### PR TITLE
EDGECLOUD-2502 IOSAppUpdate

### PR DESCRIPTION
**Pull request by ah1053**

### CODE REVIEW

File: iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift [hunk 1]
```diff
@@ -194,7 +194,7 @@ class LoginViewController: UIViewController {
             if appPortsDict.capacity == 0 {
                 throw LoginViewControllerError.runtimeError("No AppPorts in dictionary")
             }
-            // Select AppPort corresponding to internal port 3001
+            // Select AppPort corresponding to internal port 1337
```
Comment (franklin-huang-mobiledgex) [inline iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift:197]: Thanks for the fix

Comment (brucearmstrong) [reply iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift:197]: When did we switch to this elite port number?

Comment (franklin-huang-mobiledgex) [reply iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift:197]: I had been using it. the 3001 was a mistake. i have no idea why i used 1337

Comment (brucearmstrong) [reply iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift:197]: https://en.wikipedia.org/wiki/Leet

Comment (franklin-huang-mobiledgex) [reply iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift:197]: I can change the port and redeploy

Comment (brucearmstrong) [reply iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift:197]: I'm not saying there's anything wrong with that port number. I just thought it was funny that you chose it, and assumed you had done so intentionally, hence my "elite port number" comment.

**Pull request closed without merging**